mautic/mautic
# TPROD-324 - PointBundle - Migrate PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 



- `/s/points` (Actions)
  - works with and without data
  - filter
  - sort by columns
- `/s/points/new` and `/s/points/edit/{id}`
  - form renders correctly
  - Make sure each selection from **When a contact...** renders correctly
- `/s/points/triggers` (Triggers)
  - filter
  - sort by columns
- `/s/points/triggers/new` and `/s/points/triggers/edit/{id}`
  - form renders correctly
  - select events to make sure they render correctly
  - GIVEN you and editing a trigger AND it has an existing event THEN make sure when you try to edit the event that the form renders correctly
- global search will display trigger
- global search will display action

**Known Issues**
- GIVEN you are on the index/list route WHEN you click on the publish icon next to the name of the action or trigger THEN it will no longer work and must reload the page
- Global search will sometimes display an error https://github.com/mautic/mautic/issues/11977
- Cloning a trigger will not clone the events https://github.com/mautic/mautic/issues/11978
